docs/usage_guide.rst

```diff
@@ -891,6 +891,22 @@ Here the non obvious fields are:
 - ``Location Label``: This is the address related to the event, for example if you are receiving one asset in a transfer or calling a contract will match with your address.
 - ``Counterparty``: This is the other part of the transaction, the address you are interacting with. Can be a protocol identifier if the transaction is decoded as part of a protocol.
 
+If any event was not decoded the way you expected it to be, you can always customize events using the settings described above or file a bug report on our github repository / in our discord server. The customizations that you make also affect how events are processed in accounting.
+Examples of customization:
+- You can set ``Event Type`` to ``Transfer`` if you are sending money to a friend / other your account and don't want the event to be taxable.
+- You can set ``Event Type`` to ``Deposit`` / ``Withdrawal`` and ``Event Subtype`` to ``Deposit Asset`` / ``Remove Asset`` if you are depositing or withdrawing assets from an exchange or a protocol. Then this event won't be considered taxable in P&L reports. Currently rotki doesn't detect deposits / withdrawals automatically.
```

Comment:
Since you repeat it in all the cases the 

```- You can set Event Type to```

I would move it to a common line. What do you think?

Reply:
Abstracted `You can set` out. Kept `Event Type` in each line since otherwise it can be a bit confusing that `Event Type` is abstracted out while `Event Subtype` is not.